<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>565</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>EAGLe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QTreeView" name="treeView">
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QFormLayout" name="sshLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="openfilenag">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Open a .SSH file to get started...</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>990</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="openSSH"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="aboutEagle"/>
    <addaction name="aboutQt"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionEnableSSXHack"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="openSSH">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open .&amp;SSH</string>
   </property>
   <property name="toolTip">
    <string>Open a .SSH file for preview.</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionBIGF">
   <property name="text">
    <string>BIGF</string>
   </property>
  </action>
  <action name="actionBIG4_RefPack">
   <property name="text">
    <string>BIG4/C0FB (RefPack)</string>
   </property>
  </action>
  <action name="aboutEagle">
   <property name="text">
    <string>About EAGLe</string>
   </property>
  </action>
  <action name="openBig">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open .BIG (Unsupported for right now, sorry!)</string>
   </property>
  </action>
  <action name="aboutQt">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
  <action name="actionEnableSSXHack">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable SSX Mult Hack</string>
   </property>
   <property name="toolTip">
    <string>Enables a hack multiplying RGB values which seems to only be needed for SSX.</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
